model/Versioned.php

```diff
@@ -131,6 +131,14 @@ class Versioned extends DataExtension implements TemplateGlobalProvider {
 	protected static $versionableExtensions = array('Translatable' => 'lang');
 
 	/**
+	 * Permissions necessary to view records outside of the live stage (e.g. archive / draft stage).
+	 *
+	 * @config
+	 * @var array
+	 */
+	private static $non_live_permissions = array('CMS_ACCESS_LeftAndMain', 'CMS_ACCESS_CMSMain', 'VIEW_DRAFT_CONTENT');
```

Comment:
The way the check is done, this is an `or`, so any of these permissions would do?